### PR TITLE
Fix Select Not Working on Tab Key & Hide picker on focusout

### PR DESCRIPTION
**Pull request by agungjk**
I try to fix this problem like my issue, can you review it?

### CODE REVIEW

**Comment by icarusion** [comment]
I test, and it will trigger other bug in Select, when click Select, it will not be displaied

**Comment by agungjk** [comment]
@icarusion it work on filterable select, 

> I test, and it will trigger other bug in Select, when click Select, it will not be displaied

i can't reproduce it, can you show to me on gif or etc ?

**Comment by agungjk** [comment]
@icarusion i'm already reproduce it and make some bug fix

File: src/components/select/select.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div
             :class="selectionCls"
             ref="reference"
-            @click="toggleMenu">
```
Comment (SergioCrisostomo) [inline src/components/select/select.vue]: This change will make `toggleMenu` method obsolete. In this case you should also remove it.

**Comment by SergioCrisostomo** [comment]
@agungjk thanks for the pull request. Can you add a test case for the problem you are fixing please?

使用下面的在线链接可以快速创建示例 
https://codepen.io/anon/pen/KvJJdw

**Comment by agungjk** [comment]
Thank you @SergioCrisostomo 
this is test case [https://codepen.io/agungjk/pen/JrPxRx?editors=0010](https://codepen.io/agungjk/pen/JrPxRx?editors=0010)

28966871-b7fcb29e-7942-11e7-8af8-5578131fe152

I think the CI failed because i remove the obsolete function

**Comment by SergioCrisostomo** [comment]
@agungjk thank you!

This is a good idea and the TAB navigation has to be fixed. There are more components that need this change so I will let it open some days and use your work when we fix the others also.

**Pull request closed without merging**